alure.pxd

```diff
@@ -546,7 +548,34 @@ cdef extern from 'alure2.h' namespace 'alure' nogil:
         void destroy() except +
 
     cdef cppclass AuxiliaryEffectSlot:
-        pass
+        ctypedef AuxiliaryEffectSlotImpl* handle_type
+
+        AuxiliaryEffectSlot()    # nil
```

Comment:
Inline comments should be no closer than 2 spaces from the code, yet starts (count from `#`) on a 4-divisible column.